karma/test-controllers.coffee

```diff
@@ -371,18 +378,34 @@ describe('controllers:', () ->
         }, $scope.startTime, 1)
 
         fetchOld.resolve({results: [test.msg3, test.msg2], hasMore: true})
-        
-        expect($intervalSpy).toHaveBeenCalledWith($scope.autoRefresh, 10000)
 
         expect($scope.items).toEqual([test.msg3, test.msg2])
         expect($scope.oldItemsMore).toEqual(true)
         expect($scope.isInfiniteScrollEnabled()).toEqual(true)
       )
 
+      it('polls for new and updated messages ', () ->
+        fetchOld = spyOnPromise($q, $scope, MessageService, 'fetchOld')
+        $scope.poll()
+        
+        expect($scope.pollBusy).toEqual(true)
+        expect($scope.activeSearchRefresh.last_refresh).toEqual(utcdate(2016, 1, 2, 3, 0, 0, 0))
+        expect($scope.activeSearchRefresh.after).toEqual(utcdate(2016, 1, 2, 3, 0, 0, 0))
+
+        expect(MessageService.fetchOld).toHaveBeenCalledWith({
+          after: utcdate(2016, 1, 2, 3, 0, 0, 0), archived: false, before: null, contact: null, folder: 'inbox', groups: [], label: null, last_refresh: utcdate(2016, 1, 2, 3, 0, 0, 0), text: null, 
```

Comment:
The current logic on the api side appears to disregard `after` if `last_refresh` is given, which means the `after` value is no longer used. Since we provide both values here, wondering if that means we want the api to allow these params to be used together.